Frameworks/CoreText/CTRun.mm

```diff
@@ -79,101 +88,122 @@ CTRunStatus CTRunGetStatus(CTRunRef run) {
 }
 
 /**
- @Status Stub
+ @Status Interoperable
  @Notes
 */
 const CGGlyph* CTRunGetGlyphsPtr(CTRunRef run) {
-    UNIMPLEMENTED();
-    return StubReturn();
+    _CTRun* runPtr = static_cast<_CTRun*>(run);
+    return runPtr && runPtr->_glyphs.size() ? runPtr->_glyphs.data() : NULL;
```

Comment:
the docs have these specified as NULL.  is there any case where a developer could be expecting NULL and nullptr would not work?

---
In reply to: [78227167](https://github.com/Microsoft/WinObjC/pull/989#discussion_r78227167) [](ancestors = 78227167)